packages/icons/icons.json

```
@@ -4009,5 +4009,26 @@
         "tags": "model, ml",
         "group": "action",
         "codepoint": 62269
+    },
+    {
+        "displayName": "Lengthen",
+        "iconName": "lengthen",
+        "tags": "lengthen, text",
```

no need to put icon name parts ("lengthen" and "text") in the tags

```suggestion
        "tags": "long, longer, more, words, symbols",
```